### PR TITLE
CAMEL-20113 camel/tests consolidating

### PR DESCRIPTION
**Pull request by kulagaIA**
# Description

org.apache.camel.itest.jaxb:
-SpringLoadRouteFromXmlTest - moved to the camel-jaxb component;
-JaxbFallbackTypeConverterTest - moved to the camel-jaxb component;

org.apache.camel.itest.jetty:
-JettyBridgeHostHeaderIssueTest - moved to camel-jetty;
-JettyConstantSetHeaderTest - setting headers is checked in other  camel-jetty tests, so it is deleted;
-JettyFailoverRoundRobinTest - moved to camel-jetty;
-JettyJmsShutdownInProgressTest - jetty graceful shutdown is checked in org.apache.camel.component.jetty.JettySuspendWhileInProgressTest, so it is deleted;
-JettyJmsTest - test was made just to show example route, so it is deleted;
-JettyJmsTwowayTest - test was made just to show example route, so it is deleted;
-JettyRestRedirectTest - moved to camel-jetty;
-JettySimpleSetHeaderTest - setting headers is checked in other  camel-jetty tests, so it is deleted;
-JettyValidatorStreamTest - moved to camel-jetty;
-JettyValidatorTest - moved to camel-jetty;
-JettyXsltHttpTemplateTest - moved to camel-jetty;
-JettyXsltTest - moved to camel-jetty;

# Tracking
https://issues.apache.org/jira/browse/CAMEL-20113

### CODE REVIEW

File: components/camel-jaxb/src/test/java/org/apache/camel/jaxb/FallbackTypeConverterTest.java [hunk 1]
```diff
@@ -14,21 +14,21 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.camel.itest.jaxb;
-
-import java.io.InputStream;
```
Comment (orpiske) [inline components/camel-jaxb/src/test/java/org/apache/camel/jaxb/FallbackTypeConverterTest.java:19]: A note: you need to adjust your formatter to match the coding standard used by the project. Imports like this should not come last. The order should be +/-:

``
java.*
javax.* 

all other imports
```

Although much of this is done automatically by the build, it prevents us from having to re-generate the code.

File: components/camel-jaxb/src/test/java/org/apache/camel/jaxb/FallbackTypeConverterTest.java [hunk 2]
```diff
@@ -14,21 +14,21 @@
...
+package org.apache.camel.jaxb;
 
 import org.apache.camel.Message;
 import org.apache.camel.builder.RouteBuilder;
 import org.apache.camel.converter.jaxb.FallbackTypeConverter;
-import org.apache.camel.itest.jaxb.example.Bar;
+import org.apache.camel.foo.bar.Bar;
```
Comment (orpiske) [inline components/camel-jaxb/src/test/java/org/apache/camel/jaxb/FallbackTypeConverterTest.java:22]: Let's use a more descriptive package name here.

**Review by orpiske — CHANGES_REQUESTED** [review]
Once again, thanks for your contribution @kulagaIA! 

As you are contributing more often, I took the time to point a few additional things we look for in contributions, so future contributions are easier to merge and adjust to our code base. 

In addition to the items I pointed above, one more request: please, can you rebase the commits and rewrite the commit message in the format `CAMEL-20113: camel/tests consolidating" (note the 2 dots separating the ticket from the message - this ensures the commit is automatically associated with the Jira ticket). 

Marking as request changes as I am sharing some additional tips to improve future contributions.

**Comment by kulagaIA** [comment]
@orpiske thanks for taking time to guide me! Looks like everything is in order now.

**Review by orpiske — APPROVED** [review]
Thanks for applying the changes!

**Pull request merged**